core/kernel.rbs

```diff
@@ -1241,7 +1241,9 @@ module Kernel : BasicObject
   #     [0..4, 0..4, 0..4]
   #
   def self?.p: [T] (T arg0) -> T
-             | (*untyped arg0) -> Array[untyped]
+             | (untyped, untyped, *untyped) -> Array[untyped]
+             | () -> nil
+
```

Comment:
It appears that this line is the difference and the test is failing.

Reply:
Thanks, I remove it